README.md

```diff
@@ -8,6 +8,12 @@ Module add below ckeditor plugin to provide better content publish options,
   3. Drupal media library
   4. Layout manager (with optional module as it require bootstrap)
 
+Dependencies:
+------------
+This module require following ckeditor plugins, please download and place them inside libraries folder
```

Comment:
Can you please mention the path, like `This is how your directory structure should loik like - DIRECTORY/STRUCTURE`